### ISSUE TITLE
don't filter empty strings

### ISSUE THREAD

**Issue by bpappin** [NijiDigital/poesie#15]
Got this error exporting strings.

> !    Found 1 empty value(s) for the following term(s):
> !     - "empty_blank"

In this case case, I obviously know it's blank, since I named it that way.
Maybe warm the user, but the tool should not arbitrarily filter out empty strings.

**Comment by AliSoftware** [comment]
Good point. This filter removing empty strings was first added because sometimes POEditor.com returns inconsistent values in the json returned by their API including empty translations for strings that you created and some point and removed later but was still returned by the API…

Maybe it's not worth force-filtering then anymore since we recently migrated to their new v2 API which might not have that problem anymore. 

@bpappin care to do a pull request? That shouldn't be much work to remove that existing condition from the existing code 😉
